roles/edpm_libvirt/molecule/vagrant/molecule.yml

```diff
@@ -7,8 +7,8 @@ driver:
     name: libvirt
   provision: no
   parallel: true
-  # default_box: 'generic/rocky9'
-  default_box: 'generic/centos9s'
+  default_box: 'generic/rocky9'
+  # default_box: 'generic/centos9s'
```

Comment:
do we need this commented out change?

Reply:
i canged it be cause c9s package repo were down which is ironic because that's why i changed form rocky in the first place.

really either can be used provided you the package repos actually work.

so no its technically not needed bug 'generic/centos9s' was not working yesteday